### PR TITLE
add audio filter

### PR DESCRIPTION
**Pull request by timotto**
This pull request adds the capability to specify audio filters using `SetAudioFilter` as it is already possible for video filters using `SetFilter`.

### CODE REVIEW

**Comment by timotto** [comment]
As you suggested there's now a SetVideoFilter method with the SetFilter marked deprecated.

File: models/media.go [hunk 1]
```diff
@@ -499,6 +518,8 @@ func (m *Mediafile) ToStrCommand() []string {
 		"HlsSegmentDuration",
 		"HlsPlaylistType",
 		"Filter",
```
Comment (rohitggarg) [inline models/media.go]: This is not needed

File: models/media.go [hunk 2]
```diff
@@ -517,9 +538,21 @@ func (m *Mediafile) ToStrCommand() []string {
...
 
+func (m *Mediafile) ObtainAudioFilter() []string {
+	if m.audioFilter != "" {
+		return []string{"-af", m.audioFilter}
+	}
+	return nil
+}
+
+// Deprecated: Use ObtainVideoFilter instead.
 func (m *Mediafile) ObtainFilter() []string {
```
Comment (rohitggarg) [inline models/media.go]: This is not needed now

Comment (timotto) [reply models/media.go]: I removed the superfluous code

**Comment by rohitggarg** [comment]
@xfrr care to merge?

**Pull request merged**